File: src/main/java/no/nav/veilarbperson/client/pdl/domain/FamiliemedlemTilgangsstyrt.java
package no.nav.veilarbperson.client.pdl.domain;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDate;

@Data
@Accessors(chain = true)
public class FamiliemedlemTilgangsstyrt {
    private String fornavn;
    private LocalDate fodselsdato;
    private Boolean erDod;
    private Integer alder;
    private Boolean erEgenAnsatt;
    private boolean harVeilederTilgang = false;
    private String gradering; // diskresjonskode
    private RelasjonsBosted relasjonsBosted;
}

File: src/main/java/no/nav/veilarbperson/client/pdl/domain/FamiliemedlemTilgangsstyrt.kt
package no.nav.veilarbperson.client.pdl.domain

import java.time.LocalDate

data class FamiliemedlemTilgangsstyrt(
    val fornavn: String? = null,
    val fodselsdato: LocalDate? = null,
    val erDod: Boolean? = null,
    val alder: Int? = null,
    val erEgenAnsatt: Boolean? = null,
    val harVeilederTilgang: Boolean = false,
    val gradering: String? = null, //diskresjonskode
    val relasjonsBosted: RelasjonsBosted? = null
)
